bin/uglifyjs

```diff
@@ -223,7 +223,28 @@ function run() {
         }
         fatal(ex);
     } else if (program.output == "ast") {
+        if (!options.compress && !options.mangle) {
+            result.ast.figure_out_scope(typeof options.mangle == "object" ? options.mangle : {});
```

Comment:
`typeof options.mangle == "object"` can't be true here?

Reply:
(well technically it can if it's `null`, but not sure if we want to go there...)

Reply:
will fix